packages/logs/src/domain/logger.ts

```diff
@@ -40,19 +40,19 @@ export class Logger {
 
   @monitored
   log(message: string, messageContext?: object, status: StatusType = StatusType.info) {
-    this.handleLogStrategy({ message, context: deepClone(messageContext) as Context, status }, this)
+    this.handleLogStrategy({ message: sanitize(message)!, context: sanitize(messageContext) as Context, status }, this)
   }
 
   debug(message: string, messageContext?: object) {
-    this.log(message, messageContext, StatusType.debug)
+    this.log(message, messageContext as object, StatusType.debug)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** the cast seems unnecessary, ‏same for other levels
```suggestion
    this.log(message, messageContext, StatusType.debug)
```

Reply:
👌